### PR TITLE
Change score in `TrackCard` to only show on 2 decimal point

### PR DESCRIPTION
**Pull request by HelloYeew**
Just need to change this since this look too ugly and I forget to change it.

image

### CODE REVIEW

File: maisim/maisim.Game/Graphics/UserInterface/TrackCard.cs [hunk 1]
```diff
@@ -133,7 +133,7 @@ private void load(TextureStore textureStore)
                                                             {
                                                                 Anchor = Anchor.Centre,
                                                                 Origin = Anchor.Centre,
-                                                                Text = $"{score.Accuracy.ToString(CultureInfo.InvariantCulture)}%",
+                                                                Text = $"{score.Accuracy.ToString("0.00", CultureInfo.InvariantCulture)}%",
```
Comment (Game4all) [inline maisim/maisim.Game/Graphics/UserInterface/TrackCard.cs:136]: This should use o!f localisation extensions for handling culture format

**Review by Game4all — COMMENTED** [review]
Kind of a bit but sane otherwise

**Comment by Game4all** [comment]
Pushed a commit that should fix the accuracy being wrongly formatted. @HelloYeew please check it correctly formats using your locale (dot or comma, digits)

**Comment by HelloYeew** [comment]
It's show correctly to me. Also change decimal point to 4 since I am researching on it and it uses 4 decimal point on accuracy.

**Pull request merged**